torch_geometric/nn/conv/message_passing.py

```diff
@@ -99,8 +101,12 @@ def __init__(self, aggr: Optional[Union[str, List[str]]] = "add",
             assert len(set(aggr) | AGGRS) == len(AGGRS)
             self.aggr: Optional[str] = None
             self.aggrs: List[str] = aggr
+        elif isinstance(aggr, Aggregation):
```

Comment:
I think most of the `aggrs` logic can be dropped here since we now simply can utilize `MultiAggregation` for that.